### DISCUSSION TITLE
Add warnings/errors when a not enabled plugin is used within markdown template

### DISCUSSION THREAD

**Discussion by Andre601** [lowlighter/metrics#1193] [Ideas]
When using the `{%- await embed(...) %}` format alongside the markdown template is Lowlighter not checking if the required plugin(s) are enabled.

As an example, when I have this setup:
```
{%- await embed(`stats-repositories-starred`, {repositories_starred: 3}) %}
{%- await embed(`stats-repositories-random`, {repositories_random: 3}) %}
```
...but do not have the `plugin_repositories` setting in my workflow file set to `yes` / `true` will metrics still create the SVG file, resulting in an empty file being used to display stuff.

Instead of that should metrics print a warning (or even prevent the run of the workflow) about the usage of plugins that aren't enabled.

**Comment by lowlighter** [comment]
Yeah it's a good point

I'll add it to the backlog in #1196

Thanks a lot for your feedback 👍
